demo/index.html

```diff
@@ -199,8 +202,19 @@ <h2>Change book</h2>
 
       document.addEventListener('bookSearchCanceled', searchController.cancelSearch.bind(searchController));
 
-      document.addEventListener('BookReader:SearchCallback', () => {
+      document.addEventListener('BookReader:SearchCallback', (event) => {
+        const results = event.detail.results || [];
+        const rawBlob = event.detail.actual || {};
+
+        menuSlider.shadowRoot.querySelector('ia-book-search-results').results = results;
         menuSlider.shadowRoot.querySelector('ia-book-search-results').queryInProgress = false;
+
+        if (rawBlob.error) {
+          console.log('error', rawBlob);
```

Comment:
SUGGESTION: Remove debug output. Or if useful to keep, include contextual information, so it's a more "complete" error.

Reply:
updated.